apps/component-date/README.md

```diff
@@ -0,0 +1,3 @@
+Date component example
+============
+
```

Comment:
I totally agree with the intention.  I was just planning to put it in a separate PR but probably makes sense to do it now otherwise it might not happen